### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by haroon9**
Added TagsAPI example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -597,3 +598,9 @@ gitLabApi.getUserApi().createUser(userConfig, password, sendResetPasswordEmail);
 // Get a list of pages in project wiki
 List<WikiPage> wikiPages = gitLabApi.getWikisApi().getPages();
 ```
+
+#### TagsApi
+```java
+// Get a list of tags for the specified project ID
+List<Tag> tags = gitLabApi.getTagsApi().getTags(projectId);
+```
```
Comment (gmessner) [inline README.md:606]: This needs to be in alphabetical order, please move it to after the ```SystemHooksApi``` section.

Comment (haroon9) [reply README.md:606]: Ok understood.

**Comment by haroon9** [comment]
I have added the changes.

**Comment by gmessner** [comment]
@haroon9 This has been merged, thanks for your contribution.

**Pull request merged**